### PR TITLE
webui : Update homepage description

### PR DESCRIPTION
**Pull request by Yeldur**
Updated the description on our website to better convey the features of cRPG for Mount and Blade: Bannerlord.

Revised description transitions from mentioning simple character progression to highlighting engaging combat-focused gameplay, the rewards system, and the persistent character growth.

The intention behind this change is to provide a more comprehensive and appealing portrayal of our mod.

### CODE REVIEW

**Comment by Yeldur** [comment]
image

As a point of reference to how it would look.

**Review by or2e — APPROVED** [review]

**Review by verdie-g — CHANGES_REQUESTED** [review]
Could you deepl/chatgpt that to do the RU/CN translation please

File: src/WebUI/locales/en.yml [hunk 1]
```diff
@@ -742,7 +742,7 @@ onlinePlayers:
 
 homePage:
   intro: cRPG is a mod for {link}
-  description: It adds persistence to the multiplayer. You start as a peasant and you'll develop your unique character with different stats and items.
+  description: cRPG for Mount and Blade: Bannerlord transitions you from a humble peasant to a distinct warrior. Engage in gripping combat-focused gameplay, earning gold and loompoints. Use these rewards to amplify your stats and upgrade equipment. This mod subtly blends persistent character growth with riveting battles, where each fight advances your warrior's saga.
```
Comment (verdie-g) [inline src/WebUI/locales/en.yml:745]: > and loompoints

I would go with something like "experience" instead. Loompoints doesn't mean much to someone that doesn't know the game.

**Review by verdie-g — CHANGES_REQUESTED** [review]

File: src/WebUI/locales/en.yml (hunk 1 above)
Comment (Yeldur) [reply src/WebUI/locales/en.yml:745]: Good point, will update that too

**Comment by Yeldur** [comment]
I messed up the last commit header, meant it to say (RU) given that it was the RU update but meh, it was a small change so minor issue as it still conveys what change is being made

File: src/WebUI/locales/ru.yml [hunk 2]
```diff
@@ -739,7 +739,7 @@ onlinePlayers:
 
 homePage:
   intro: cRPG - это мод для {link}
-  description: Он добавляет постоянство в многопользовательскую игру. Вы начинаете как крестьянин и будете развивать своего уникального персонажа с различными статами и предметами.
+  description: cRPG для Mount and Blade: Bannerlord превращает вас из скромного крестьянина в отчетливого воина. Участвуйте в захватывающем игровом процессе, основанном на сражениях, зарабатывая золото и опыт. Используйте эти награды для повышения своих характеристик и улучшения снаряжения. Этот мод тонко сочетает постоянный рост персонажа с захватывающими сражениями, где каждый бой способствует развитию саги о вашем воине.
```
Comment (or2e) [inline src/WebUI/locales/ru.yml:742]: cRPG для Mount and Blade: Bannerlord превращает вас из скромного крестьянина в выдающегося воина. Участвуйте в захватывающем игровом процессе, ориентированном на сражениях, зарабатывая золото и опыт. Используйте эти награды для повышения своих характеристик и улучшения снаряжения. Этот мод тонко сочетает постоянный рост персонажа с захватывающими сражениями, где каждый бой продолжает сагу о вашем воине.

**Comment by Yeldur** [comment]
Have updated with changes (ty <3)

**Review by or2e — APPROVED** [review]

**Review by verdie-g — APPROVED** [review]

**Pull request merged**